### PR TITLE
[Fix - ADF 249]: UDIR GenerisSearch: User gets "get?ClassUri" cancelled request in browser console with error pop-up message after clicking 'Clear' in Modal View

### PR DESCRIPTION
**Pull request by habemuscode**
**PR to be merged before this one**

https://github.com/oat-sa/tao-core-ui-fe/pull/269

**Issue**

https://oat-sa.atlassian.net/browse/ADF-249

**Description**

Whenever the user is clicking the 'Clear' button in Modal View and then closes the Modal window, there's a loading process on the page with the common loading bar under the panel with tabs 'Items', 'Tests' etc. 
During this loading, the page is unresponsive to any interaction. After the timeout of 30 seconds, there's an error pop-up message appears and canceled the "get?ClassUri" request in the networking tab of DevTools.

**Steps to reproduce**

- Go to the Items/Tests/Assets tab.
- Click the Search icon.
- In opened Modal view window click the 'Clear' button.
- Close the Modal view.

**Actual result**

The page is irresponsive during loading and after the timeout of 30 seconds, there's an error pop-up message that appears and canceled the "get?ClassUri" request in the networking tab of DevTools.

**Expected result**

The modal window is closed and the page is available to interact with, no errors appear.

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "@oat-sa/tao-core-libs": "0.4.3",
         "@oat-sa/tao-core-sdk": "^1.10.1",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.22.8",
+        "@oat-sa/tao-core-ui": "git://github.com/oat-sa/tao-core-ui-fe#fix/ADF-249/cancel-call-if-no-elasticsearch",
```
Comment (gabrielfs7) [inline views/package.json:16]: Just to not forget to replace to proper version before merge to develop :)

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by adriloma — APPROVED** [review]
Remeber to replace @oat-sa/tao-core-ui version to `1.22.9` on `package.json` before merging.

**Comment by habemuscode** [comment]
Temporary deployed here:  http://test-as-acl-fix.playground.kitchen.it.taocloud.org:44291

**Review by 404passionFruit — APPROVED** [review]

**Comment by 404passionFruit** [comment]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable) - tested on Kitchen
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] bumping versions are applied (if applicable)

**Pull request merged**